perception/lidar_apollo_segmentation_tvm/include/lidar_apollo_segmentation_tvm/feature_map.hpp

```diff
@@ -45,6 +45,7 @@ struct FeatureMapInterface
   virtual void initializeMap(std::vector<float> & map) = 0;
   virtual void resetMap(std::vector<float> & map) = 0;
   explicit FeatureMapInterface(int32_t _channels, int32_t _width, int32_t _height, int32_t _range);
+  virtual ~FeatureMapInterface() = default;
```

Comment:
Is there a reason to have something different between here and in https://github.com/autowarefoundation/autoware.universe/pull/7759/files#diff-f161427533ca4d885ecaa95822c35e97febda3bce39f2ba4342f01d0395fd4d6R96 ?

Reply:
My apologies. My mistake. There is no reason. I fixed it!

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/7762/commits/4b29942c76c3182a4db1eda9509f7b4686ed051d